src/vstart.sh

```diff
@@ -608,8 +616,8 @@ prepare_conf() {
         enable experimental unrecoverable data corrupting features = *
         osd_crush_chooseleaf_type = 0
         debug asok assert abort = true
-$msgr_conf
-$extra_conf
+        $(format_conf "${msgr_conf}")
+        $(format_conf "${extra_conf}")
```

Comment:
It also need to format extra_conf in other sections. But I am doubt there is really need extra_conf in other sections?

Reply:
> It also need to format `extra_conf` in other sections. 

done.

> But I am doubt there is really need `extra_conf ` in other sections?

guess adding the `extra_conf` to all sections helps to override the options in the dedicated sections in case some of the options are also specified explicitly in the non-global section.